app/src/main/res/layout/list_tasks_fragment.xml

```diff
@@ -0,0 +1,179 @@
+        >
+
+        <LinearLayout
+            android:id="@+id/new_task"
+            android:layout_width="match_parent"
+            android:layout_height="93.5dp"
+            android:background="@color/reddish_pink"
+            android:orientation="vertical"
+            >
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="34.5dp"
+                android:gravity="center_vertical"
+                android:layout_marginEnd="25dp"
+                android:layout_marginStart="25dp"
+                android:layout_marginTop="5dp"
+                android:orientation="horizontal"
+                >
+
+                <EditText
+                    android:id="@+id/title_edit_text"
+                    android:layout_width="300dp"
+                    android:layout_height="wrap_content"
+                    android:background="@android:color/transparent"
+                    android:hint="@string/add_title"
+                    android:inputType="textNoSuggestions"
+                    android:layout_marginEnd="25dp"
+                    android:maxLines="1"
+                    android:nextFocusForward="@+id/description_edit_text"
+                    android:textColor="@color/white"
+                    android:textSize="16sp"
+                    android:textStyle="bold"
+                    />
+
+                <ImageView
+                    android:id="@+id/close"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="20dp"
+                    android:src="@drawable/icon_close"
+                    />
+
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="34.5dp"
+                android:gravity="center_vertical"
+                android:layout_marginBottom="5dp"
+                android:layout_marginEnd="25dp"
+                android:layout_marginStart="25dp"
+                android:orientation="horizontal"
+                >
+
+                <EditText
+                    android:id="@+id/description_edit_text"
+                    android:layout_width="300dp"
+                    android:layout_height="wrap_content"
+                    android:background="@android:color/transparent"
+                    android:hint="@string/add_description"
+                    android:inputType="textNoSuggestions"
+                    android:layout_marginEnd="25dp"
+                    android:textColor="@color/white"
+                    android:textSize="14sp"
+                    />
+
+                <TextView
+                    android:id="@+id/add_task"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:clickable="true"
```

Comment:
Note that if you set a click listener to a view, `clickable` is already set. This seems not to be necessary.